docs/static/ls-to-cloud.asciidoc

```diff
@@ -0,0 +1,68 @@
+[[connecting-to-cloud]]
+=== Sending data to Elasticsearch Service
+
+Our hosted Elasticsearch Service is available on AWS, GCP, and Azure.
+{ess-trial}[You can try the Elasticsearch Service for free].
+
+Logstash comes with two settings that simplify sending data to
+https://cloud.elastic.co/[Elastic Cloud]: Cloud ID and Cloud Auth. 
+
+[[cloud-id]]
+==== Cloud ID
+
+{ls} uses the Cloud ID, found in the Elastic Cloud web console, to build the
+Elasticsearch and Kibana hosts settings. It is a base64 encoded text value of
+about 120 characters made up of upper and lower case letters and numbers.
+If you have several Cloud IDs, you can add a label, which is ignored
+internally, to help you tell them apart. To add a label you should prefix your
+Cloud ID with a label and a `:` separator in this format "<label>:<cloud-id>"
+
+Cloud ID overwrites these settings:
```

Comment:
I would consider removing this overwrites section: `var.elasticsearch` and `var.kibana` only really relates to modules, which are deprecated IIRC, and keeping this as general information about cloud.id